### PR TITLE
Snack bar component added.

### PR DESCRIPTION
**Pull request by rutu14**
Added Snack bar component

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
- Good selection of class name
- Clean UI

Good to merge!

File: documentation/css/style.css [hunk 1]
```diff
@@ -213,10 +215,13 @@
     height: 30rem;
 }
 
+.li-code{
+    margin: 8px 0;
+}
```
Comment (bhushannaikaroskar) [inline documentation/css/style.css:220]: I think this tag can be named better, its a suggestion tough no need to change right away

**Review by bhushannaikaroskar — APPROVED** [review]
Rest all I think its good to merge

**Review by dev-prabhat — APPROVED** [review]
UI is clean , it looks good to me

File: documentation/css/style.css (hunk 1 above)
Comment (rutu14) [reply documentation/css/style.css:220]: Sure.

**Pull request merged**